adapter/scripts/build.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```

Comment:
Shall we keep -x here ? So that we could clearly see at which point this stopped executing?

Reply:
+1